Tests/LibSQL/TestSqlBtreeIndex.cpp

```diff
@@ -4,8 +4,6 @@
  * SPDX-License-Identifier: BSD-2-Clause
  */
 
-#include <unistd.h>
```

Comment:
Each of these files use `unlink` which comes from unistd.h. It's fine to remove their usage in favor of `Core::System::unlink`, but just removing the header isn't accomplishing anything, and isn't needless as long as the LibC `unlink` is used here.

Reply:
Lemme just drop this commit too. It came from the earlier version of PR where I was also removing `unistd.h` includes.